R/cfbd_players.R

```diff
@@ -300,7 +303,7 @@ cfbd_player_usage <- function(year = most_recent_cfb_season(),
     "team" = team,
     "conference" = conference,
     "position" = position,
-    "athleteID" = athlete_id,
+    "playerId" = athlete_id,
```

Comment:
might be worth flagging this in the changelog / updating the docs for this change

Reply:
This is an internal only change, end-users will not have any changes other than the intended functionality when using the parameters. I suppose it could be added to the NEWS.md